user_guide_src/source/libraries/cors/002.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Config;
+
+use CodeIgniter\Config\Filters as BaseFilters;
+
+// ...
+
+class Filters extends BaseFilters
+{
+    // ...
+    public array $filters = [
+        // ...
+        'cors' => [
```

Comment:
Is there a way to use  a different configuration (exactly like` ['filter' => 'cors:api']` )? If yes, please document.

Reply:
It is documented in the Filters page.
https://codeigniter4.github.io/CodeIgniter4/incoming/filters.html#filter-arguments

Reply:
Added a link. https://github.com/codeigniter4/CodeIgniter4/pull/8649/commits/3a7e88048c1414760c4848ff0ae8f90ff7d2b203